web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -2094,19 +2096,25 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
           />
           <FormGroup label="Extra filter">
             <JsonInput
-              value={restFilter}
-              onChange={f => {
+              value={newFilterValue}
```

Comment:
I think this should be `newFilterValue || restFilter` otherwise the filter disappears after you click Apply

Reply:
I am manually clearing the filter by setting newFilter to undefined. I think I can just remove that.